qds_sdk/cloud/cloud.py

```diff
@@ -5,3 +5,9 @@ def create_parser(self, argparser):
 
     def set_cloud_config_from_arguments(self, arguments):
         return NotImplemented
+
+    def set_composition_arguments(self, comp_group):
+        pass
```

Comment:
should we

```suggestion
        return NotImplemented
```
?

Reply:
If we go with that approach, we have to check for cloud when we set the composition. 
Rather than having an if condition to check whether the cloud supports, each cloud can set empty data or implement the function will sets the value. 
I implemented it this way to avoid that if conditions. In future, user has to implement the function and no need to edit the condition to allow cloud to set compositions.